dom/index.js

```diff
@@ -110,14 +110,20 @@ function insertBefore (target, el) {
   return once(() => parent.removeChild(el))
 }
 
+function insertChild (target, el) {
```

Comment:
_Petty Comment_: Should we call this `appendChild` to clarify the location of the insertion?

Reply:
okay